lib/views/changed-files-count-view.js

```diff
@@ -28,17 +28,13 @@ export default class ChangedFilesCountView extends React.Component {
   }
 
   render() {
-    const label =
-      (this.props.changedFilesCount === 1)
-        ? '1 file'
-        : `${this.props.changedFilesCount} files`;
     return (
       <button
         ref="changedFiles"
         className="github-ChangedFilesCount inline-block"
         onClick={this.handleClick}>
-        <Octicon icon="diff" />
-        {label}
+        <Octicon icon="git-commit" />
```

Comment:
I think changing the icon from `diff` to `git-commit` isn't a good choice. The pane is about changed files, it tells you how many files have local changes, yet it uses the symbol for commits. When I first saw the new layout, I'd expected to get the equivalent of `git log` and were confused about what the number is going to tell me.

Reply:
The Git panel includes different things:
- Changes
- Commit message editor
- Recent commits

The "Recent commits" is some sort of mini `git log`:

![commits](https://user-images.githubusercontent.com/378023/49263826-62ce7b80-f48f-11e8-9dc8-47b819b84225.gif)

So it doesn't seem easy to find a good icon that fits everything. Using the git icon should hint at: "Create and view commits".

Reply:
I'd agree, that for the pane the git commit icon is a better fit, but I'm talking about the status bar button.

The issue is, you're optimizing this for people who aren't even using it. And in doing so, you are breaking the current UI for the users you already have. You shouldn't take that lightly. The new layout doesn't convey more information, nor does it free up precious space in the status bar. Once you found out that the Git(Hub) pane exists and that the buttons will open those panes, these buttons take up space because they got bulky. 

I agree that the "n files" label wasn't perfect. What I'd suggest is merely splitting up the buttons. Make one button with just the diff icon and the number of changed files as the label. And a second button with the git-commit icon and git as its label. Both buttons open the diff/commit git pane on click, but the information is logically split. Plus, users who already discovered the git pane can hide the git button without losing functionality. Additionally, all buttons should have a tooltip like every other thing in the status bar does.
I'd even make a PR for that if you like that idea.

Reply:
> nor does it free up precious space in the status bar.
> What I'd suggest is merely splitting up the buttons. Make one button with just the diff icon and the number of changed files as the label. And a second button with the git-commit icon and git as its label.

Here a mockup:

![after](https://user-images.githubusercontent.com/378023/49708537-5f977480-fc73-11e8-938c-4645458bf13a.png)

But wouldn't two buttons take up even more space than just a single button?

![before](https://user-images.githubusercontent.com/378023/49708536-5f977480-fc73-11e8-8a1a-0c73b8cdde0f.png)

Also since the two buttons do the **same** thing, could that be confusing? People would click on both and then think something must be broken since in both cases it opens the same panel. Maybe this could be made less confusing if the focus moves to the changes and the recent commits. Like this:

![screen shot 2018-12-10 at 12 13 23 pm](https://user-images.githubusercontent.com/378023/49708915-22cc7d00-fc75-11e8-911e-9a1b6ec7b219.png)

But having the focus in two different places seems less useful for users that use the mouse to open the git panel.